.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: python
 python: 3.6
 cache: pip
-before_install:
-- pip install -U pre-commit
-- pip install pylint-django==0.11.1
-- pip install Django==2.0.5
+before_install: pip install -r requirements/test.txt
```

Comment:
Please use a sequence instead of a string, it makes diffs nicer.

Reply:
@webknjaz Sorry, I think I do not fully understand what I need to do.

Reply:
you did it already, but also taught lines, which didn't really change anything.